lib/adapters/QUnitAdapter.js

```diff
@@ -5,18 +5,52 @@ import {Test, Suite} from "../Data.js";
 export default class QUnitAdapter extends EventEmitter {
     constructor() {
         super();
-        QUnit.done(this.onDone.bind(this));
+        QUnit.config.reorder = false;
```

Comment:
Let's remove this here and keep it in the tests. And create or update a QUnit issue with details of this. It's really a QUnit bug.